govuk-one-login/tech-docs
# Ap286 remove social security claim

[AP-286](https://govukverify.atlassian.net/browse/AP-286)
## Why

No longer required

## What

Remove https://vocab.account.gov.uk/v1/socialSecurityRecord claim from examples and documentations

## Technical writer support

Do you need a tech writer's support, for example to review your PR? - **YES**

## How to review

- check out the branch 
- preview locally using docker script
- ensure there are no mentions of the https://vocab.account.gov.uk/v1/socialSecurityRecord claim

## Changelog

thus pr contains an update to the changelog

## Confirm

- [xx] Where there is any overlap I have updated or opened a PR for corresponding changes


[AP-286]: https://govukverify.atlassian.net/browse/AP-286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
